### PR TITLE
Create user2020tutorial.Rmd

### PR DESCRIPTION
**Pull request by yabellini**
Blog post user2020 tutorials AfricaR, MiR and R-Ladies

### CODE REVIEW

**Comment by maelle** [comment]
Just a small comment after skimming through the post, more URLs would be great (how to read more about MiR and AfricaR, where to find info about the tutorials etc).

**Comment by yabellini** [comment]
This texts was write for each organization and has the agreement of each one of them.
I can ask for this details at userR Slack

**Comment by yabellini** [comment]
@maelle, I just update the post with the links you suggest. Thank you!

File: content/post/user2020tutorial.Rmd [hunk 1]
```diff
@@ -58,7 +58,7 @@ Some of the R-Ladies Global initiatives to support its community are:
...
 * R-Ladies directory: A directory of R-Ladies speakers for conferences and events. See https://rladies.org/directory/ if you want to be listed or browse it.
 
-We thank all tutorial leads that graciously agreed to teach their tutorials online. useR! 2020 tutorials will be held for free in July featuring outstanding tutorial leads from around the world. Check out the tutorials page to see all tutorials that will be held in July. 
+We thank all tutorial leads that graciously agreed to teach their tutorials online. useR! 2020 tutorials will be held for free in July featuring outstanding tutorial leads from around the world. Check out the [tutorials](https://docs.google.com/spreadsheets/d/1qaQ6dARwMjAJ2oG0KZwzMvY4fJmXsSz_i-dPcg3dUmE/edit?usp=sharing) page to see all tutorials that will be held in July. 
 
 If you are an AfricaR RUG organizer who wants to host one of these tutorials, please reach out to africarusers@gmail.com, or DM Anicet Ebou (@anicetebou) on twitter or in the R users group organizer Slack. If you are an R-Ladies chapter organizer who wants to host a tutorial, you will find further information on how to do this in R-Ladies Global organizer Slack.
```
Comment (maelle) [inline content/post/user2020tutorial.Rmd:63]:
```suggestion
If you are an AfricaR RUG organizer who wants to host one of these tutorials, please reach out to africarusers@gmail.com, or DM [Anicet Ebou (@anicetebou) on Twitter](https://twitter.com/anicetebou) or in the R users group organizer Slack. If you are an R-Ladies chapter organizer who wants to host a tutorial, you will find further information on how to do this in R-Ladies Global organizer Slack.
```

Also, the link to the RUG organizers slack could be useful.

File: content/post/user2020tutorial.Rmd [hunk 2]
```diff
@@ -14,7 +14,7 @@ output: html_document
 knitr::opts_chunk$set(echo = TRUE)
 ```
 
-We are extremely excited to announce that organizers from AfricaR, the MiR’s community, and R-Ladies will facilitate the online tutorial sessions for useR! 2020. In addition to taking deep dives into R topics from the tutorial experts, useR! 2020 attendees will have the opportunity to meet inclusive community organizers and learn more about the communities they serve. We hope to increase the representation in attendance of members from these inclusive communities within the tutorial sessions. 
+We are extremely excited to announce that organizers from [AfricaR](https://africa-r.org), the [MiR’s community](https://medium.com/@doritolay/introducing-mir-a-community-for-underrepresented-users-of-r-7560def7d861), and [R-Ladies](https://rladies.org/) will facilitate the online tutorial sessions for useR! 2020. In addition to taking deep dives into R topics from the tutorial experts, useR! 2020 attendees will have the opportunity to meet inclusive community organizers and learn more about the communities they serve. We hope to increase the representation in attendance of members from these inclusive communities within the tutorial sessions.
```
Comment (maelle) [inline content/post/user2020tutorial.Rmd:17]:
```suggestion
We are extremely excited to announce that organizers from [AfricaR](https://africa-r.org), the [MiR’s community](https://medium.com/@doritolay/introducing-mir-a-community-for-underrepresented-users-of-r-7560def7d861), and [R-Ladies](https://rladies.org/) will facilitate the online tutorial sessions for [useR! 2020](https://user2020.r-project.org/). In addition to taking deep dives into R topics from the tutorial experts, useR! 2020 attendees will have the opportunity to meet inclusive community organizers and learn more about the communities they serve. We hope to increase the representation in attendance of members from these inclusive communities within the tutorial sessions. 
```

**Comment by yabellini** [comment]
We decided to publish the blog post on the AfricaR web page. I'm going to close this PR.

**Pull request closed without merging**